modules/UI/side_pannels/SidePanelToggler.js

```diff
@@ -86,7 +86,7 @@ function toggle (object, selector, onOpenComplete,
         // before it's actually visible.
         // (Chrome seems to return {true} for an element which is in the DOM and
         // has non-null size set).
-        $(selector).css("width", "20%");
+        $(selector).css("width", "200px");
```

Comment:
Hi Jana, because on small screens 20% of 320px is 64px the chat panel was too narrow and thus unusable on small phone screen for anything (input nor reading) so I rather put fixed value of 200px there instead.